### DISCUSSION TITLE
How i do change my name

### DISCUSSION THREAD

**Discussion by Vector-69** [community/community#18994] [Other Feature Feedback, Questions, & Ideas] [answered]
i want to to change my name

**Comment by ApidriuC** [comment, marked as answer]
https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-personal-account-settings/changing-your-github-username

**Comment by webVueBlog** [comment]
更改您的 GitHub 用户名

- 在任何页面的右上角，单击您的个人资料照片，然后单击设置。
- 在左侧边栏中，单击Account。
- 在“更改用户名”部分，单击更改用户名。
- 阅读有关更改用户名的警告。如果您仍想更改您的用户名，请单击我了解，让我们更改我的用户名。
- 键入新用户名。
- 如果您选择的用户名可用，请单击更改我的用户名。如果您选择的用户名不可用，您可以尝试其他用户名或您看到的建议之一。
